integration_test.go

```diff
@@ -64,20 +63,20 @@ func testCartographerPosition(t *testing.T, svc slam.Service, useIMU bool, expec
 		}
 
 	case runtime.GOOS == "darwin" && useIMU:
-		expectedPos = r3.Vector{X: 4.4700878707562035, Y: 3.1781587655776358, Z: 0}
+		expectedPos = r3.Vector{X: 3.2858556935949954, Y: 2.707920249449911, Z: 0}
 		expectedOri = &spatialmath.R4AA{
-			RX:    0.9861776038047263,
-			RY:    0.1637212678758259,
-			RZ:    0.025477052402116784,
-			Theta: 0.02399255141454847,
+			RX:    0.985674239521596,
```

Comment:
[note] These values got updated when I changed the timing of the IMU by 1 millisecond. This change was necessary to support online testing with IMU (before we were doing offline but to support localization mode we had to mvoe to online mode).